app/views/declaration_information.scala.html

```diff
@@ -70,72 +70,98 @@
+                        ),
+                        classes = "submission__ucr"
                     ),
-                    value = Value(
-                        content = submission.mrn.map(HtmlContent(_)).getOrElse(Empty)
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("submissions.lrn"))
+                        ),
+                        value = Value(
+                            content = HtmlContent(submission.lrn)
+                        ),
+                        classes = "submission__lrn"
                     ),
-                    classes = "submission__mrn"
-                )
-            ).filterNot(_.value.content == Empty)
-        ))
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("submissions.mrn"))
+                        ),
+                        value = Value(
+                            content = submission.mrn.map(HtmlContent(_)).getOrElse(Empty)
+                        ),
+                        classes = "submission__mrn"
+                    )
+                ).filterNot(_.value.content == Empty)
+            ))
 
-        @renderDeclarationLinks
+            @renderDeclarationLinks
+        </div>
 
         @if(notifications.nonEmpty) {
-            <div class="govuk-grid-column" id ="declarationHistory">
-                @govukTable(Table(
-                    rows = notifications.sorted.reverse.zipWithIndex.map { case (notification, index) =>
-                        Seq(
-                            TableRow(
-                                content = HtmlContent(SubmissionStatus.format(notification.status)),
-                                attributes = Map("id" -> s"notification_status_$index")
-                            ),
-                            TableRow(
-                                content = HtmlContent(ViewDates.formatDateAtTime(notification.dateTimeIssuedInUK)),
-                                attributes = Map("id" -> s"notification_date_time_$index")
-                            ),
-                            TableRow(
-                                content = HtmlContent(errorOrUploadFileLink(submission.uuid, notification.status)),
-                                attributes = Map("id" -> s"notification_action_$index")
+
+            <div class="govuk-grid-column-full">
+                <h2 class="govuk-heading-m">@messages("submissions.timeline")</h2>
+
+                @if(hasDmsdocNotifications) {
+                    <p id="content-on-dmsdoc" class="govuk-body">
+                        @messages("submissions.content.on.dmsdoc")
+                        @if(sfusConfig.isSfusSecureMessagingEnabled) {
+                            @Html(messages("submissions.content.on.dmsdoc.sfus", sfusInboxLink))
+                        }
+                    </p>
+                }
+
+                <div class="govuk-grid-column">
```

Comment:
Nit: Out of curiosity: how does this class work? What is the width of the element?